scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -530,6 +532,8 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             File.Delete(compilersDeps);
             File.Delete(compilersRuntimeConfig);
 
+            CrossgenUtil.CrossgenDirectory(c, outputDir);
```

Comment:
@eerhardt I like to keep stage1 and stage2 similar since we sort of use it for testing CLI itself.   There are plenty of other optimizations we can do
- crossgen in parallel
- run tests in parallel, actually was experimenting with this. The time for test execution is down to less than 5 mins on my Macbook Pro.